.github/workflows/deploy-documentation.yml

```diff
@@ -15,28 +15,18 @@ jobs:
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
     steps:
-      - name: Check out code
+      - name: Checkout code
         uses: actions/checkout@v4
-        with:
-          fetch-depth: 2
```

Comment:
Shouldn't we keed the `fetch-depth`? This might speed up a little bit the pipeline isn't it?

Reply:
You are right, I'll restore it!

Reply:
How about keeping the default value, which is `1`?

From [@actions/checkout documentation](https://github.com/actions/checkout?tab=readme-ov-file#usage):
> Number of commits to fetch. 0 indicates all history for all branches and tags.
> Default: 1

From the looks of it we don't need to fetch commits more than 1 commit. 
I don't see any operation requiring git history, but I might be wrong.

Reply:
Oh I didn't know that the default is `1`. That's even better!